src/source/view/controls/Styles.bs

```diff
@@ -157,4 +157,123 @@ namespace mv.styles
       }
     }
   end function
+
+  function getDefaultTextInputStyle()
```

Comment:
please remove these 2 functions - we will lean into the style manager now -so for example, the style sin maestroDefaultStyles.json